hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -221,6 +229,15 @@ newtype PropertyName =
       unPropertyName :: String
     } deriving (Eq, Ord, Show, IsString, Semigroup, Lift)
 
+-- | The acceptable occurrence of false positives
+--
+--   Example, @Confidence 10^9@ would mean that you'd accept a false positive
+--   for 1 in 10^9 tests.
+newtype Confidence =
+  Confidence {
+    unConfidence :: Integer
```

Comment:
Do you envision using this with confidence greater than `10^18`? (`Int64`)

Reply:
QuickCheck gives a pretty good explanation of this in their docs:

> If you are using checkCoverage as part of a test suite, you should be careful not to set certainty too low. If you want, say, a 1% chance of a false positive during a project's lifetime, then certainty should be set to at least 100 * m * n, where m is the number of uses of cover in the test suite, and n is the number of times you expect the test suite to be run during the project's lifetime. The default value is chosen to be big enough for most projects.

So for instance - let's say we're writing a big project with 100 ocurrences of `cover`, then we'd get:

```
100 * 100 * n = confidence
```

Let's say that we expect to run the project 20 times per day per developer with 5 developers. That would make it:

```
100 * 100 * (20 * 5 * daysWithoutFalsePositives) = confidence
<=>
1 000 000 * daysWithoutFalsePositives = confidence
```

Let's say we want 5 years with 1% chance of false positives:

```
1 000 000 * daysWithoutFalsePositives = confidence
<=>
1 000 000 * 365 * 5 = 1 825 000 000
```

Which is veeeeeeery roughly `2*10^9`. So yeah, maybe not. Would you rather use `Int64`?

Reply:
Yeah we're talking age of the universe scales at 10^18.